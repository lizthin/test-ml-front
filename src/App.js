import React from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import Items from "./pages/Items/Items.js";

/**
 * App funtion router call default route, scalable switch router flow
 * @param {object} props routers component
 */
const App = (props) => {
  return (
    <>
      <Router>
        <div>
          <Switch>
            <Route path="/" component={Items} history={props.history} />
          </Switch>
        </div>
      </Router>
    </>
  );
};
export default App;
