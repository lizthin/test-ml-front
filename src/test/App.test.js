import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Test app router', () => {
    test('should display <App />', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    })
});