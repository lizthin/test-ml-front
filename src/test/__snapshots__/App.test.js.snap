// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app router should display <App /> 1`] = `
<Fragment>
  <BrowserRouter>
    <div>
      <Switch>
        <Route
          component={[Function]}
          path="/"
        />
      </Switch>
    </div>
  </BrowserRouter>
</Fragment>
`;
